src/main/webapp/content/scss/global.scss

```diff
@@ -1087,3 +1091,15 @@ body.transparent-background {
 .sidebar-collapsed {
     display: none;
 }
+
+.fs-small {
```

Comment:
Great idea to add global classes for these font sizes. However, please use either rem values or the corresponding keywords like `small` or `smaller` and so on. rem is relative to the font size of the root element. In our case 1 rem corresponds to 16px. The keywords correspond to the following rem values:

xx-small: 0.625rem
x-small: 0.75rem
small: 0.875rem
medium: 1rem
large: 1.125rem
x-large: 1.5rem
xx-large: 2rem
xxx-large: 3rem